ui/packages/catalog-ui-search/src/main/webapp/component/map-context-menu/map-context-menu.view.js

```diff
@@ -188,6 +188,14 @@ module.exports = Marionette.LayoutView.extend({
     this.attachListenersToClipboard(
       new Clipboard(this.el.querySelector('.interaction-copy-dms'))
     )
+    if (this.options.mapModel.get('clickMgrs')) {
+      this.attachListenersToClipboard(
+        new Clipboard(this.el.querySelector('.interaction-copy-mgrs'))
```

Comment:
✏️ see if you can just use the mapModel's clickMgrs attribute for this

Reply:
OBE since the rework using React.